codespell_lib/data/dictionary.txt

```diff
@@ -7068,6 +7069,7 @@ magnatiude->magnitude
 magnatude->magnitude
 magnificient->magnificent
 magolia->magnolia
+mailinglist->mailing
```

Comment:
this should be removed.

Reply:
For me the right spell is "mailing list".
Anyway, I remove it.